src/modules/components/vote/voteProposalDataListItem/voteProposalDataListItemStructure/voteProposalDataListItemStructure.tsx

```diff
@@ -0,0 +1,52 @@
+import classNames from 'classnames';
+import { DataList, Tag, type IDataListItemProps } from '../../../../../core';
+import { voteIndicatorToTagVariant } from '../../voteDataListItem/voteDataListItemStructure/voteDataListItemStructure';
+
+export interface IVoteProposalDataListItemStructureProps extends IDataListItemProps {
+    /**
+     * The ID of proposal.
+     */
+    id: string;
+    /**
+     * The account details of the voter.
+     */
+    title: string;
+    /**
+     * The vote of the user.
+     */
+    voteIndicator: 'yes' | 'no' | 'abstain' | 'approve';
+    // TO-DO: implement formatter decision
+    /**
+     * The Unix timestamp of the transaction.
+     */
+    date?: string;
+}
+
+export const VoteProposalDataListItemStructure: React.FC<IVoteProposalDataListItemStructureProps> = (props) => {
+    const { title, id, voteIndicator, date, className, ...otherProps } = props;
+
+    return (
+        <DataList.Item className={classNames('flex items-center gap-x-3 md:gap-x-4', className)} {...otherProps}>
+            <div className="flex w-full flex-col gap-y-1 md:gap-y-1.5 md:text-lg">
```

Comment:
We can probably avoid having another wrapping `div` and just apply the classnames to the data-list-item component